service/src/main/java/bio/terra/workspace/service/resource/controlled/flight/clone/workspace/CreateIdsForFutureStepsStep.java

```diff
@@ -18,21 +18,13 @@ public class CreateIdsForFutureStepsStep implements Step {
   @Override
   public StepResult doStep(FlightContext context) throws InterruptedException, RetryException {
     FlightMap workingMap = context.getWorkingMap();
-    workingMap.put(
-        ControlledResourceKeys.CREATE_GCP_CLOUD_CONTEXT_FLIGHT_ID,
```

Comment:
I will want to remove these keys from `ControlledResourceKeys`… will do so in the cleanup PR.